qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -427,11 +427,15 @@ def pos_perc(self):
         return self._pos_perc
 
     def to_perc(self, x=None, y=None):
-        js_code = javascript.assemble('scroll', 'to_perc', x, y)
+        js_code = javascript.assemble('scroll', 'to_perc', x, y,
+                                      config.val.scrolling.smooth)
         self._tab.run_js_async(js_code)
 
     def to_point(self, point):
-        js_code = javascript.assemble('window', 'scroll', point.x(), point.y())
+        # FIXME to_point will try to go to the same point point even in
```

Comment:
I implemented frame scrolling for this function, but this makes marks scroll in-frame, which could mean that jump-mark might be unintuitive when a page has mutliple frames. I'm not sure what the correct thing to do here is, should we use the root window always for jump-mark?